pom.xml

```diff
@@ -444,14 +507,6 @@ Without it Maven would put two versions of each class file into the jars
         <artifactId>target-versions-plugin</artifactId>
         <version>${tycho.version}</version>
       </plugin>
-      <plugin>
-        <groupId>org.eclipse.tycho</groupId>
-        <artifactId>target-platform-configuration</artifactId>
-        <version>${tycho.version}</version>
-        <configuration>
-          <executionEnvironment>JavaSE-1.8</executionEnvironment>
-        </configuration>
-      </plugin>
       <plugin>
```

Comment:
Das beunruhigt mich etwas, warum hast du das rausgenommen? Merci

Reply:
Du hast den Finger auf einen wunden Punkt gelegt. Wenn ich diese Konfiguration (welche ich für sinnvolle hate), deaktiviere habe ich im repository und im erzeugten product zwei verschiedene jars z.b. 
    -rw-r--r-- 1 niklaus niklaus 217990 Jun 25 13:45 ./ch.elexis.core.p2site/target/repository/plugins/ch.rgw.utility_3.7.0.20180622-0722.jar
    -rw-r--r-- 1 niklaus niklaus 219673 Jun 25 13:45 ./ch.elexis.core.p2site/target/products/ch.elexis.core.application.ElexisApp/linux/gtk/x86/plugins/ch.rgw.utility_3.7.0.20180622-0722.jar
Im repository findet man das erzeugte Jar, im target, dasjenige aus https://download.elexis.info/elexis/master/p2/elexis-3-core/plugins/ch.qos.logback.slf4j_1.0.7.v201505121915.jar

Ich versuche mal, ob ich nicht das repository und das product in zwei verschiedenen Projekten bauen muss.